tests/ui/default_numeric_fallback_f64.rs

```diff
@@ -0,0 +1,172 @@
+// aux-build:macro_rules.rs
+
+#![warn(clippy::default_numeric_fallback)]
+#![allow(unused)]
+#![allow(clippy::never_loop)]
+#![allow(clippy::no_effect)]
+#![allow(clippy::unnecessary_operation)]
+#![allow(clippy::branches_sharing_code)]
+
+#[macro_use]
+extern crate macro_rules;
+
+mod basic_expr {
+    fn test() {
+        // Should lint unsuffixed literals typed `f64`.
+        let x = 0.12;
+        let x = [1., 2., 3.];
+        let x = if true { (1., 2.) } else { (3., 4.) };
+        let x = match 1. {
+            _ => 1.,
+        };
```

Comment:
NIT: This triggers the `clippy::match_single_binding` lint. Could you allow this lint to keep the file error files to only contain `default_numeric_fallback` lint triggers?

```txt
error: this match could be replaced by its body itself
  --> $DIR/default_numeric_fallback_f64.rs:19:17
   |
LL |           let x = match 1. {
   |  _________________^
LL | |             _ => 1.,
LL | |         };
   | |_________^ help: consider using the match body instead: `1.`
   |
   = note: `-D clippy::match-single-binding` implied by `-D warnings`
```

Reply:
Good catch! I fixed that.